### PR TITLE
doc: add minutes for meeting 12 dec 2018

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2018-12-12.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  * Workers - Gabriel has been working with Anna to better integrate support for
+    native modules. Allows for module unloading, remaining blocking issue is crash on
+    AIX, Gabriel investigating.
+
+  * Governance - we started a project board to track responsibilities of various
+    stakeholders within the proposed foundation. We will be using this artifact to
+    help determine how many Top-Level committees there will be and what they
+    will be responsible for. https://github.com/nodejs/bootstrap/projects/1
+
+  * Open Web Standards https://github.com/nodejs/node/pull/21551
```
Comment (MylesBorins) [inline meetings/2018-12-12.md]: likely need a space between title and the link

File: meetings/2018-12-12.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+* cli: add --max-http-header-size flag [#24811](https://github.com/nodejs/node/pull/24811)
+  * Added to agenda for visibility.
+  * Plan is to land and backport sooner than later as people were affected by change in the
+    community release and this provides a work around.
+  * Removed from agenda for now.
+
+* Fixing child_process module to check values passed strictly to the options object [#24267](https://github.com/nodejs/node/pull/24267)
+  * controversial as there is disagreement in the issue as to whether this should land.
+  * Back to github and invite Liran to next TSC meeting to provide a deeper dive and then
+    we can decide on next steps.
```
Comment (Trott) [inline meetings/2018-12-12.md]: This is a disappointing result. This doesn't seem like a discussion/decision that needs to happen synchronously (and therefore necessarily exclude people).

Comment (Trott) [reply meetings/2018-12-12.md]: Meh, I suppose a case can be made to seed the broader policy discussion with a conversation first. Seems to suggest a lack of engagement to me, to be honest, but it's not unreasonable. I'm going to delete my comment.

File: meetings/2018-12-12.md (hunk 1 above)
Comment (mhdawson) [reply meetings/2018-12-12.md]: @MylesBorins fixed.

**Pull request merged**